crates/router/src/connector/riskified/transformers/api.rs

```diff
@@ -0,0 +1,595 @@
+    tracking_numbers: String,
+    tracking_urls: Option<String>,
+}
+
+impl TryFrom<&frm_types::FrmFulfillmentRouterData> for RiskifiedFullfillmentRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &frm_types::FrmFulfillmentRouterData) -> Result<Self, Self::Error> {
+        Ok(Self {
+            order: OrderFullfillment {
+                id: item.attempt_id.clone(),
+                fulfillments: FulfilmentData {
+                    fulfillment_id: item.payment_id.clone(),
+                    created_at: common_utils::date_time::now(),
+                    status: item
+                        .request
+                        .fulfillment_req
+                        .fulfillment_status
+                        .clone()
+                        .and_then(get_fulfillment_status),
+                    tracking_company: item
+                        .request
+                        .fulfillment_req
+                        .tracking_company
+                        .clone()
+                        .ok_or(errors::ConnectorError::MissingRequiredField {
+                            field_name: "tracking_company",
+                        })?,
+                    tracking_numbers: item.request.fulfillment_req.tracking_number.clone().ok_or(
+                        errors::ConnectorError::MissingRequiredField {
+                            field_name: "tracking_numbers",
+                        },
+                    )?,
+                    tracking_urls: item.request.fulfillment_req.tracking_url.clone(),
+                },
+            },
+        })
+    }
+}
+
+impl
+    TryFrom<
+        ResponseRouterData<
+            Fulfillment,
+            RiskifiedFulfilmentResponse,
+            frm_types::FraudCheckFulfillmentData,
+            frm_types::FraudCheckResponseData,
+        >,
+    >
+    for types::RouterData<
+        Fulfillment,
+        frm_types::FraudCheckFulfillmentData,
+        frm_types::FraudCheckResponseData,
+    >
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: ResponseRouterData<
+            Fulfillment,
+            RiskifiedFulfilmentResponse,
+            frm_types::FraudCheckFulfillmentData,
+            frm_types::FraudCheckResponseData,
+        >,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            response: Ok(frm_types::FraudCheckResponseData::FulfillmentResponse {
+                order_id: item.response.order.id,
+                shipment_ids: Vec::new(),
+            }),
+            ..item.data
+        })
+    }
+}
+
+#[derive(Debug, Deserialize, Serialize, Eq, PartialEq, Clone)]
+pub struct ErrorResponse {
+    pub error: ErrorData,
+}
+
+#[derive(Debug, Deserialize, Serialize, Eq, PartialEq, Clone)]
+pub struct ErrorData {
+    pub message: String,
+}
+
+impl TryFrom<&api_models::payments::Address> for OrderAddress {
+    type Error = error_stack::Report<errors::ConnectorError>;
```

Comment:
we can define Error type globally once, since it's same across every implementations